ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/components/StatusCountIcon.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Icon } from '@patternfly/react-core';
+import { BarsIcon, CheckCircleIcon, SecurityIcon } from '@patternfly/react-icons';
+import pluralize from 'pluralize';
+
+import IconText from 'Components/PatternFly/IconText/IconText';
+
+type StatusCountIconProps = {
+    text: string;
+    status: 'pass' | 'fail' | 'other';
+    count: number;
+};
+
+function getStatusIcon(status, count) {
+    let color = 'var(--pf-v5-global--disabled-color--100)';
+    if (count > 0) {
+        switch (status) {
+            case 'fail':
+                color = 'var(--pf-v5-global--danger-color--100)';
+                break;
+            case 'pass':
+                color = 'var(--pf-v5-global--success-color--100)';
+                break;
+            case 'other':
+                color = 'var(--pf-v5-global--disabled-color--100)';
+                break;
+            default:
+                break;
+        }
+    }
+
+    switch (status) {
+        case 'fail':
+            return <SecurityIcon color={color} />;
+        case 'pass':
+            return <CheckCircleIcon color={color} />;
+        case 'other':
+            return <BarsIcon color={color} />;
+        default:
+            return null;
+    }
+}
+
+function StatusCountIcon({ text, status, count }: StatusCountIconProps) {
```

Comment:
Are you suggesting to separate this into 3 separate components? `PassStatusCountIcon`, `FailStatusCountIcon`, and `OtherStatusCountIcon`?

Reply:
I was commenting more about the naming of the component. We might have other components that are status counts too. It might be better to specify what kind of status count this is. It might make it easier to search as well.